### DISCUSSION TITLE
Comparison to existing tools

### DISCUSSION THREAD

**Discussion by hansbogert** [holos-run/holos#380] [Q&A] [answered]
I think it would be beneficial to include a comparison page to the documentation. Personally I'd like to know how this project compares to Timoni [1]. 

[1]: https://timoni.sh/

**Comment by jeffmccune** [comment]
Thanks for the suggestion, we’ll have one up soon.

In the meantime take a look at: https://news.ycombinator.com/item?id=42382206

As it relates to Timoni the biggest difference is it is a controller that applies manifests to the cluster. Holos is a command line tool that renders manifests to the file system.

In practice this means Holos pairs nicely with the rendered manifests pattern and integrates with ecosystem tools like Kargo.

**Reply by jeffmccune** [reply, marked as answer]
We've published a comparison page at https://holos.run/docs/comparison/
